src/scroll-timeline-base.js

```diff
@@ -310,63 +431,6 @@ export class ScrollTimeline {
     return data.scrollOffsets;
   }
 
-  set startScrollOffset(offset) {
-    if (offset == "auto")
-      offset = AUTO;
-    let currentStlOptions = scrollTimelineOptions.get(this);
-    // Allow extensions to override scroll offset calculation.
-    currentStlOptions.startScrollOffsetFunction = null;
-    for (let i = 0; i < extensionScrollOffsetFunctions.length; i++) {
-      let result = extensionScrollOffsetFunctions[i].parse(offset);
-      if (result !== undefined) {
-        offset = result;
-        currentStlOptions.startScrollOffsetFunction =
-          extensionScrollOffsetFunctions[i].evaluate;
-        break;
-      }
-    }
-    if (offset != AUTO && !scrollTimelineOptions.get(this).startScrollOffsetFunction) {
-      let parsed = parseLength(offset);
-      // TODO: This should check CSSMathSum values as well.
-      if (!parsed || (parsed instanceof CSSUnitValue && parsed.unit == "number"))
-        throw TypeError("Invalid start offset.");
-    }
-    currentStlOptions.startScrollOffset = offset;
-    updateInternal(this);
```

Comment:
I think we need a call to updateInternal in the setter for scrollOffsets now that it has an effect on the current time.